source/voice_control/voice_remote_local_assistant.markdown

```
@@ -77,3 +95,10 @@ View some of the options in the video below. Explained by Mike Hansen, creator o
 <lite-youtube videoid="Tk-pnm7FY7c" videoStartAt="1589" videotitle="Configure your local Assist pipeline for your setup"></lite-youtube>
 
 The options are also documented in the add-on itself. Go to the {% my supervisor_addon addon="core_whisper" title="**Whisper**" %} or the {% my supervisor_addon addon="core_piper" title="**Piper**" %} add-on and open the **Documentation** page.
+
+Also be sure to check the specific tutorial for [using Piper in Automations](voice_control/using_tts_in_automation/)
+
+## Next steps
+
+
```

Markdownlint (0.35.0)

Expected: 1; Actual: 2
Multiple consecutive blank lines

(MD012, no-multiple-blanks)